lls/main.c

```diff
@@ -1086,6 +1257,9 @@ lls_stage2(void *arg)
 		if (ret < 0)
 			return ret;
 		ret = register_nd_router_sol_or_adv_acl_rules(&net_conf->back);
+		if (ret < 0)
+			return ret;
+		ret = register_ping_acl_rules(&net_conf->back);
```

Comment:
This code should go to the `if` statement of ARP, this is the `if` statement of ND.